src/calculateRentalCost.js

```diff
@@ -5,6 +5,19 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
+  const x = price * days;
+  if (days >= 7) {
+    return  x - 50;
```

Comment:
change all the Magic numbers
```suggestion
    return  basePrice - LONG_TERM_DISCOUNT;
```